onmt/modules/LSTM.lua

```diff
@@ -104,8 +116,22 @@ function LSTM:_buildLayer(inputSize, hiddenSize)
   local x = inputs[3]
 
   -- Evaluate the input sums at once for efficiency.
-  local i2h = nn.Linear(inputSize, 4 * hiddenSize)(x)
-  local h2h = nn.Linear(hiddenSize, 4 * hiddenSize)(prevH)
+  local i2h = nn.Linear(inputSize, 4 * hiddenSize, false)(x)
```

Comment:
for reference, we know that there is a redundancy here - but we added it for being equivalent with cudnn.LSTM. When batch-normalization will be used - the module will not be anymore compatible with cudnn.LSTM anyway, so in that case you can make the change, but let us manage to keep the non-batch-normalization mode as is.